datasets/simulated.py

```diff
@@ -1,7 +1,5 @@
-from benchopt import BaseDataset, safe_import_context
-
-with safe_import_context() as import_ctx:
-    import numpy as np
+from benchopt import BaseDataset
+from benchopt.datasets import make_correlated_data
```

Comment:
This should be imported in the `safe_import_context` to avoid slowing down auto completion due to importint `numpy`.
```suggestion
with safe_import_context() as import_ctx:
    from benchopt.datasets import make_correlated_data
```